CatRentalMachine.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iomanip>
+#include "CatRentalMachine.h"
+
+CatRentalMachine::CatRentalMachine(Cat& rentalCat):
+cat(rentalCat)
```

Comment:
From Project 5.3:
```
/*
If you aren't modifying the passed-in object inside the function, pass by 'const reference'.
Marking a function parameter as 'const' means that you are promising that the parameter will not be modified.
```
`rentalCat` is not being modified in this constructor, so it should be marked `const`
- [ ] revise